packages/ui/components/form/select/select.stories.mdx

```diff
@@ -1,6 +1,6 @@
 import { Canvas, Meta, Story, ArgsTable } from '@storybook/addon-docs';
 import { Examples, Example, Note, Title,CustomArgsTable, VariantRow,VariantsTable} from '@calcom/storybook/components'
-import { Icon } from "@calcom/ui/Icon";
+import { Icon } from "@calcom/ui";
```

Comment:
Icon is breaking stories when directly imported - speaking to @JeroenReumkens this has been fixed in a PR he is working on